.github/release-drafter.yml

```diff
@@ -1,27 +1,31 @@
-name-template: '$RESOLVED_VERSION: Version title TODO change me'
+name-template: '$RESOLVED_VERSION: Version title (TODO: set meaningful title)'
 tag-template: '$RESOLVED_VERSION'
 categories:
   - title: 'Breaking changes'
-    label: 'PR:breaking'
+    label: 'PR type: breaking'
   - title: 'Features'
-    label: 'PR:feature'
+    label: 'PR type: feature'
   - title: 'Bug Fixes'
-    label: 'PR:bugfix'
+    label: 'PR type: bugfix'
   - title: 'Miscellaneous'
-    label: 'PR:miscellaneous'
+    label: 'PR type: miscellaneous'
 change-template: '- $TITLE (#$NUMBER)'
 version-resolver:
   default: minor
   major:
-    label: 'PR:major'
+    label: 'PR scope: major'
   minor:
-    label: 'PR:minor'
+    label: 'PR scope: minor'
   patch:
-    label: 'PR:incremental'
+    label: 'PR scope: incremental'
 exclude-labels:
 - 'PR:skip-changelog'
```

Comment:
I don't see this label in the list of labels. Don't you want to create it?

Reply:
![Screenshot from 2020-06-11 10-03-43](https://user-images.githubusercontent.com/1457576/84360898-2246d680-abcb-11ea-9501-ac96e487e984.png)

Reply:
![Screenshot from 2020-06-11 10-06-34](https://user-images.githubusercontent.com/1457576/84361032-58845600-abcb-11ea-859a-20097c7a6a6b.png)

Reply:
AAAAaaaaaah your comment was pointing at the line BELOW the diff :o

Ok !

Reply:
Yes it was about `PR:skip-changelog`. Sorry for the confusion. And I am not sure I can create labels myself.

Reply:
I think you can. I've created it.

![Screenshot from 2020-06-11 11-14-45](https://user-images.githubusercontent.com/1457576/84367608-c84b0e80-abd4-11ea-83bb-17fb3d3fae9f.png)